.circleci/config.yml

```diff
@@ -77,6 +77,12 @@ anchors:
       # Workbench release tag format example: v5-3-rc1
       only: /^v[0-9]+-[0-9]+-rc[0-9]+$/
 
+  # Job runs for PR branches only
+  filter_only_pr_branch: &filter_only_pr_branch
+    tags:
+      ignore: /.*/
```

Comment:
I don't  understand what this does. Is it ignoring hidden files (like a glob), or is this a regex (so  it's  ignoring everything)?

Reply:
Igore all tag branches (include release tag branches).